src/components/DataBar/DataBar.tsx

```diff
@@ -0,0 +1,197 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './DataBar.module.css';
+
+import DataBarSegment from '../DataBarSegment';
+import type { Variants } from '../DataBarSegment';
+
+type DataBarSegmentProps = {
```

Comment:
it's kinda confusing to call this `Props` because it doesn't fully match the actual Prop types. could it be `Segment` or `DataBarSegmentObject` instead?

Reply:
Yup makes sense, will change to `Segment`